templates/reset_password/reset.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Reset your password{% endblock %}
+
+{% block body %}
+<h1>Reset your password</h1>
+
+{{ form_start(resetForm) }}
+{{ form_row(resetForm.plainPassword) }}
+
+<button class="btn btn-primary">Reset password</button>
+{{ form_end(resetForm) }}
```

Comment:
And indent here

Reply:
fixed in maker
- [ ] ensure fixed in result